packages/commonwealth/server/routes/createChain.ts

```diff
@@ -429,6 +432,10 @@ const createChain = async (
     });
 
     chain.Contract = contract;
+
```

Comment:
I don't think `createChain.ts` is used anymore. We should just delete this file.

Reply:
deleted